Entities/Special/CTF/CTF_Flag.cfg

```diff
@@ -5,7 +5,7 @@
 $sprite_factory                                   = generic_sprite
 @$sprite_scripts                                  = CTF_Flag.as;
 $sprite_texture                                   = CTF_Flag.png
-s32_sprite_frame_width                            = 8
```

Comment:
Why did you change this and the frame number below? Frame sizes should be powers of 2

Reply:
Why is this? I modified the width of the flag sprite to align better with the box2d shape. Made it easy to draw the progress bar in the middle.

Reply:
non-power of 2 can cause rendering issues on older graphics drivers/cards. we can modify the sprite_offset_x or move the flag 1 pixel over in the spritesheet, so it's centered in it's frame. Those would be alternatives.